pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -717,7 +717,7 @@ def test_tdi_add_overflow(self):
             )
 
         # These should not overflow!
-        exp = TimedeltaIndex([NaT])
+        exp = TimedeltaIndex([NaT], dtype="m8[ns]")
```

Comment:
What's the default unit that would be used in this case? 
Still nanoseconds?

Reply:
just updated with the fix+test for this